docs/en-US/component/calendar.md

```diff
@@ -47,14 +47,14 @@ Note, date time locale (month name, first day of the week ...) are also configur
 
 ## Attributes
 
-| Name                    | Description                                                                                                                                                    | Type           | Default |
-| ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | ------- |
-| `model-value / v-model` | binding value                                                                                                                                                  | `Date`         | —       |
-| `range`                 | time range, including start time and end time. Start time must be start day of week, end time must be end day of week, the time span cannot exceed two months. | `[Date, Date]` | —       |
+| Name                    | Description                                                                                                                                                    | Type                   | Default |
+| ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- | ------- |
+| `model-value / v-model` | binding value                                                                                                                                                  | ^[Date]                | —       |
```

Comment:
`model-value / v-model` -> model-value / v-model

The same below. Consistent with other documents.

Reply:
Right, my mistake for missing these.